### PR TITLE
Add a SuggestionsReview util

### PR DESCRIPTION
**Pull request by phlax**
to allow mass accept/reject of suggestions, with email notifications and adding comments on unit

### CODE REVIEW

File: pootle/templates/editor/email/suggestions_accepted_with_comment.txt [hunk 1]
```diff
@@ -0,0 +1,10 @@
+{% load i18n %}
+{% filter wordwrap:70 %}{% blocktrans %}The following suggestions submitted by you for have been accepted with the following comment:{% endblocktrans %}{% endfilter %}
```
Comment (unho) [inline pootle/templates/editor/email/suggestions_accepted_with_comment.txt]: by you for have?

File: pootle/templates/editor/email/suggestions_rejected_with_comment.txt [hunk 2]
```diff
@@ -0,0 +1,10 @@
+{% load i18n %}
+{% filter wordwrap:70 %}{% blocktrans %}The following suggestions submitted by you for have been rejected with the following comment:{% endblocktrans %}{% endfilter %}
```
Comment (phlax) [inline pootle/templates/editor/email/suggestions_rejected_with_comment.txt]: ditto

File: pootle/apps/pootle_store/utils.py [hunk 3]
```diff
@@ -0,0 +1,102 @@
...
+            from_email=None,
+            recipient_list=[suggester.email],
+            fail_silently=True)
+
+    def notify_suggesters(self, rejected=True, comment=""):
+        for suggester, suggestions in self.users_and_suggestions.items():
+            if rejected:
+                self.notify_rejected(suggester, suggestions, comment)
+            else:
+                self.notify_accepted(suggester, suggestions, comment)
```
Comment (unho) [inline pootle/apps/pootle_store/utils.py]: `notify_accepted` is basically repeating `notify_rejected` but changing the subject and the template. Perhaps it is better to get the subject and template based on the `rejected` value and have a single method to send email.

File: pootle/apps/pootle_store/utils.py [hunk 4]
```diff
@@ -0,0 +1,102 @@
...
+                     user=self.reviewer)).save()
+
+    def accept_suggestions(self):
+        for suggestion in self.suggestions:
+            suggestion.unit.accept_suggestion(
+                suggestion,
+                suggestion.unit.store.translation_project,
+                self.reviewer)
+
+    def accept(self, comment=""):
```
Comment (unho) [inline pootle/apps/pootle_store/utils.py]: Where is `reject` method?

Comment (unho) [inline pootle/apps/pootle_store/utils.py]: Addressed

Comment (unho) [inline pootle/apps/pootle_store/utils.py]: This has been explained on chat.

**Comment by unho** [comment]
lgtm

**Pull request merged**